src/core_plugins/kibana/public/management/react/index-pattern/index-pattern-view/index-pattern-view.container.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { IndexPatternView as IndexPatternViewComponent } from './index-pattern-view.component';
+
+import {
+  CustomProps
+} from 'plugins/kibana/management/react/lib/custom_props';
+
+import {
+  TabProps
+} from 'plugins/kibana/management/react/lib/tab_props';
+
+import {
+  refreshFields,
+  deleteIndexPattern,
+  setDefaultIndexPattern,
+  fetchIndexPattern,
+} from 'plugins/kibana/management/react/store/actions';
+
+import {
+  getSelectedIndexId,
+  getSelectedIndexFields,
+  getSelectedIndexTimeFieldName,
+  getSelectedIndexIsDefault,
+} from 'plugins/kibana/management/react/store/reducers';
```

Comment:
Why the mixing of selectors in reducers? Is this something you're planning to do in the "final version" or just a shortcut for now? If it's a long-term design decision, I'd be interested in knowing why.